docs/Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.4.0.9000
```

Comment:
Please update this to say `1.5 (in progress)`, which is the convention we've used in the past.

Reply:
Change made.